hw6_epam_traning/src/main/java/com/mapLine/MapCreator.java

```diff
@@ -0,0 +1,30 @@
+package com.mapLine;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class MapCreator {
+
+    public MapCreator() {
```

Comment:
Do you need empty constructor? And just fyi when you create a class the empty constructor is created by default